openwisp_controller/config/api/views.py

```diff
@@ -289,6 +302,58 @@ def certificate_delete_invalidates_cache(cls, organization_id, common_name):
         cls.get_device_group.invalidate(cls, org_slug, common_name)
 
 
+class RevisionListView(ProtectedAPIMixin, AutoRevisionMixin, ListAPIView):
+    serializer_class = ReversionSerializer
+
+    def get_queryset(self):
+        model_slug = self.kwargs.get('model').lower()
+        return (
+            Version.objects.select_related('revision')
+            .filter(content_type__model=model_slug)
+            .order_by('-revision__date_created')
+        )
```

Comment:
The view name contains Revision, but the queryset is returning `Version` objects. This is confusing. 

If I understand correctly, a revision is made up of multiple versions (from different objects). 

Is it logically possible to restore a particular version of the object while others are unaffected? Could this potentially render the database in a broken state?